bugbug/models/qaneeded.py

```diff
@@ -96,6 +96,16 @@ def get_labels(self):
                             for label in ["qawanted", "qe-verify", "qaurgent"]
                         ):
                             classes[bug_id] = 1
+
+            if not found_qa:
```

Comment:
No need to check the condition again and loop through the changes again, just add the inner if (the one about the change) in the earlier loop